extensions/wikia/ArticleVideo/bower_components/html5-skin/js/components/scrubberBar.js

```diff
@@ -0,0 +1,326 @@
+      if (this.touchInitiated){
+        evt = evt.touches[0];
+      }
+      var deltaX = evt.clientX - this.lastScrubX;
+      var scrubbingPlayheadX = this.props.currentPlayhead * this.state.scrubberBarWidth / this.props.duration + deltaX;
+      this.props.controller.updateSeekingPlayhead((scrubbingPlayheadX / this.state.scrubberBarWidth) * this.props.duration);
+      this.setState({
+        scrubbingPlayheadX: scrubbingPlayheadX
+      });
+      this.lastScrubX = evt.clientX;
+    }
+  },
+
+  handlePlayheadMouseUp: function(evt) {
+    if (!this.isMounted()) {
+      return;
+    }
+    this.props.controller.startHideControlBarTimer();
+    evt.preventDefault();
+    // stop propagation to prevent it from bubbling up to the skin and pausing
+    evt.stopPropagation(); // W3C
+    evt.cancelBubble = true; // IE
+
+    this.lastScrubX = null;
+    if (!this.touchInitiated){
+      ReactDOM.findDOMNode(this).parentNode.removeEventListener("mousemove", this.handlePlayheadMouseMove);
+      document.removeEventListener("mouseup", this.handlePlayheadMouseUp, true);
+    }
+    else{
+      ReactDOM.findDOMNode(this).parentNode.removeEventListener("touchmove", this.handlePlayheadMouseMove);
+      document.removeEventListener("touchend", this.handlePlayheadMouseUp, true);
+    }
+    this.props.controller.seek(this.props.currentPlayhead);
+    if (this.isMounted()) {
+      this.setState({
+        currentPlayhead: this.props.currentPlayhead,
+        scrubbingPlayheadX: 0
+      });
+      this.props.controller.endSeeking();
+    }
+    this.touchInitiated = false;
+  },
+
+  handleScrubberBarMouseDown: function(evt) {
+    if (this.props.controller.state.screenToShow == CONSTANTS.SCREEN.AD_SCREEN) return;
+    if (evt.target.className.match("oo-playhead")) { return; }
+    if (this.touchInitiated && evt.type === "mousedown") { return; }
+    var offsetX = 0;
+    this.touchInitiated = evt.type === "touchstart";
+    if (this.touchInitiated){
+      offsetX = evt.targetTouches[0].pageX - evt.target.getBoundingClientRect().left;
+    }
+    else {
+      offsetX = evt.nativeEvent.offsetX == undefined ? evt.nativeEvent.layerX : evt.nativeEvent.offsetX;
+    }
+
+    this.setState({
+      scrubbingPlayheadX: offsetX
+    });
+    this.props.controller.updateSeekingPlayhead((offsetX / this.state.scrubberBarWidth) * this.props.duration);
+    this.handlePlayheadMouseDown(evt);
+  },
+
+  handleScrubberBarMouseOver: function(evt) {
+    if (!this.props.skinConfig.controlBar.scrubberBar.thumbnailPreview) return;
+    if (this.props.controller.state.screenToShow == CONSTANTS.SCREEN.AD_SCREEN) return;
+    if (this.isMobile) { return; }
+    if (evt.target.className.match("oo-playhead")) { return; }
+
+    this.setState({
+      hoveringX: evt.nativeEvent.offsetX
+    });
+  },
+
+  handleScrubberBarMouseMove: function(evt) {
+    this.handleScrubberBarMouseOver(evt);
+  },
+
+  handleScrubberBarMouseOut: function(evt) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove the unused function parameter "evt". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AUnusedFunctionArgument)